### DISCUSSION TITLE
Hold wns resulting infinite

### DISCUSSION THREAD

**Discussion by khaledhasanzami** [The-OpenROAD-Project/OpenROAD#3014] [General]
I was running the riscv model with default setup. When I was trying to find wns for hold after the full run (or in cts, route). It is returning as INF.

Command:
```
report_worst_slack -min
```
Returns `worst slack INF`

**pdk:** asap7
**design:** riscv32i

Any idea why is that and how to solve it?

**Comment by maliberty** [comment]
In which PDK?  How did you load the design?  Please give specific steps to reproduce the issue.

**Reply by khaledhasanzami** [reply]
pdk: asap7
design: riscv32i

Loaded the design with `make gui_6_finale.odb`
I am adding the infos in the main question too.

**Comment by maliberty** [comment]
I guess you meant ```make gui_6_final.odb``` without the 'e'.  I get:

```
>>> report_worst_slack -min
worst slack 40.29
```

What commit id are you using for OR & ORFS?  This would have worked better as an issue with the required info.

**Reply by khaledhasanzami** [reply]
Thank you. Pulling the updated version worked. I was using the previous commit earlier. I am no filing an issue beacuse I am not sure of the commit I was using earlier.
